src/mesh/vtk_helper.jl

```diff
@@ -115,6 +119,96 @@ function quad_vtk_order(corner_verts, order, dim, skip = false)
+
+Compute the coordinates of a VTK_LAGRANGE_wedge of a wedge of order order 
+defined by the coordinates of the vertices given in corner_verts. dim is the
+dimension of the coordinates given. If skip is set to true, the coordinates
+of the vertex- and edge-points aren't computed, which can be used to compute
+points of a VTK_LAGRANGE_WEDGE
+Inspired by: https://github.com/ju-kreber/paraview-scripts/blob/master/node_ordering.py
+"""
+function wedge_vtk_order(corner_verts, order, dim)
+    
+    @assert order >= 0 "`order` must be non-negative."
+    
+    coords = copy(corner_verts)
+    if order == 0
+        return coords
+    end
+    num_verts_on_edge = order - 1
+    edges = SVector((1,2), (2,3), (3,1), (4,5), (5,6), (6,4), (1,4), (2,5), (3,6))
+    for (frm, to) in edges
+        tmp = n_verts_between(num_verts_on_edge, corner_verts[:, frm], corner_verts[:, to])
+        tmp_vec = Vector{Float64}(undef, dim)
+        for i in 1:num_verts_on_edge
+            for j in 1:dim
+                tmp_vec[j] = tmp[j][i]
+            end
+            coords = hcat(coords, tmp_vec)
+        end
+    end
+    #faces
+    tri_faces = SVector((1,2,3), (4,5,6))
+    quad_faces = SVector((1,2,5,4), (2,3,6,5), (1,3,6,4))
+    #triangular faces
+    for indices in tri_faces
+        tri_nodes = Matrix{Float64}(undef,dim,0)
+        tmp_vec = Vector{Float64}(undef, dim)
+        for j in indices
+            tri_nodes = hcat(tri_nodes,corner_verts[:,j])
+        end
+        face_coords = triangle_vtk_order(tri_nodes, order, 3, true)
+        face_coords = sort_by_axis(face_coords)
+        if length(face_coords) > 0
+            for i in range(1,size(face_coords)[2])
+                for j in 1:dim
+                    tmp_vec[j] = face_coords[j,i]
+                end
+                coords = hcat(coords, tmp_vec)
+            end
+        end
+    end
+    #quadrilateral faces
+    for indices in quad_faces
+        tmp_vec = Vector{Float64}(undef, dim)
+        quad_nodes = Matrix{Float64}(undef, dim,0)
+        for j in indices
+            quad_nodes = hcat(quad_nodes, corner_verts[:,j])
+        end
+        face_coords = quad_vtk_order(quad_nodes, order, 3, true)
+        if length(face_coords) > 0
+            for i in 1:size(face_coords)[2]
+                for j in 1:dim
+                    tmp_vec[j] = face_coords[j,i]
+                end
+                coords = hcat(coords, tmp_vec)
+            end
+        end
+    end
+    #volume
+    e_z = (corner_verts[:,4] - corner_verts[:,1]) ./ order
+    interior_tri_verts = [corner_verts[:,1] corner_verts[:,2] corner_verts[:,3]]
+    face_coords = triangle_vtk_order(interior_tri_verts, order, 3, true)
+    face_coords = sort_by_axis(face_coords)
+    for i in range(1,num_verts_on_edge)
+        tmp_vec = Vector{Float64}(undef, dim)
+        face_coords = face_coords .+ e_z
+        if length(face_coords) > 0
+            for k in range(1,size(face_coords)[2])
```

Comment:
```suggestion
            for k in 1:size(face_coords, 2)
```
For consistency (unless there's a reason you need `range`?)